lib/middleman-hashicorp/hashicorp.rb

```diff
@@ -365,19 +370,30 @@ def arch_for_filename(path)
       end
 
       #
-      # Return a page's path on GitHub, relative to its source
-      # repo's root directory.
+      # Return the GitHub URL associated with the project
+      # @return [String] the project's URL on GitHub
+      # @return [false] if github_slug hasn't been set
+      #
+      def github_url(specificity = :repo)
+        return false if github_slug.nil?
+        base_url = 'https://www.github.com/' + github_slug
```

Comment:
If you switch to double quotes, you could also interpolate instead of creating extra strings